src/main/java/io/appium/java_client/service/local/AppiumServiceBuilder.java

```diff
@@ -401,6 +403,17 @@ protected ImmutableList<String> createArgs() {
         return new ImmutableList.Builder<String>().addAll(argList).build();
     }
 
+    @Override
+    public AppiumDriverLocalService build() {
+        File driverExecutable = ReflectionHelpers.getPrivateFieldValue(
+                AppiumServiceBuilder.class, this, "exe", File.class
```

Comment:
`AppiumServiceBuilder.class` -> `org.openqa.selenium.remote.service.DriverService.Builder.class` - it seems this should fix the access since the field `exe` is private

Reply:
already done ;)